### PR TITLE
test: Stop notification for conformance testing

### PR DESCRIPTION
**Pull request by engedaam**
Fixes #N/A 

**Description**
- Stop notification for conformance testing

**How was this change tested?**
- Tested inside of forked repo 

**Does this change impact docs?**
- [] Yes, PR includes docs updates
- [] Yes, issue opened: # 
- [x] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/e2e-conformance-trigger.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ jobs:
   conformance:
     if: github.repository == 'aws/karpenter' || github.event_name == 'workflow_dispatch'
     strategy:
+      max-parallel: 1
```
Comment (jonathan-innis) [inline .github/workflows/e2e-conformance-trigger.yaml:10]: Will this cause timeouts?

Comment (engedaam) [reply .github/workflows/e2e-conformance-trigger.yaml:10]: It should not timeout. By default, each conformance job will have 6 hours to complete. The workflow has 35 days before timeout.

Comment (jonathan-innis) [reply .github/workflows/e2e-conformance-trigger.yaml:10]: Ok, sounds good. I'd be more apt to increase our API limits first before restricting this parallelism. 1 is going to mean that all of these jobs are going to take a really long time to run so I think we should do a little validation to tweak the value to something that makes sense after we get our limits/quotas bumped.

**Review by jonathan-innis — APPROVED** [review]
🚀

**Pull request merged**